darwin/torch/dataset.py

```diff
@@ -26,19 +26,36 @@ def get_dataset(
     """
     Pulls a dataset from Darwin and returns a Dataset class that can be used with a PyTorch dataloader
 
-    Input:
-        dataset_name: Identifier of the dataset in Darwin
-        image_set: Split set [train, val, test]
-        mode: selects the dataset type [image_classification, instance_segmentation, semantic_segmentation]
-        transforms: List of PyTorch transforms
-        client: Darwin's client
-        val_percentage: percentage of images used in the validation set
-        test_percentage: percentage of images used in the validation set
-        force_fetching: discard local dataset and pull again from Darwin
-        force_resplit: discard previous split and create a new one
-        split_seed: fix seed for random split creation
-
-    Output:
+    Parameters
```

Comment:
The numpy docstring notation goes by:
```
Parameters
----------
[variable_name] : [variable_type]
       [description]
index : int 
       example of description

Returns
--------
[variable_name] : [variable_type]
[description]
```
The advantage of using it is that then automated tools (such as Pycharm) auto-detect issues between methods signatures and documentation (in this case extra parameters in doc string). I updated the docstring with some TODOs highlighting the missing parts and issues.

Reply:
I also fixed that in a later commit